### DISCUSSION TITLE
How to sync Saleor core configuration across environments

### DISCUSSION THREAD

**Discussion by madmonkeyworks** [saleor/saleor#14115] [Deployment]
Hi there, 

We have Saleor deployed in different environments: 
- dev (Saleor cloud)
- stage (AWS)
- production (AWS)

In order to test the stage, we need to make sure that all the configuration is the same as in production. I'm talking about the configuration of warehouses, taxes, shipping methods, categories etc. Not only the settings – for example Shipping zone > default shipping weight unit – but also metadata, and even our custom Saleor apps configuration (metadata etc.)

Is there a way to export the configuration from one environment and import it in the other? 
Or any programmatic way to set the selected configuration on Saleor start?

Thanks

**Comment by timuric** [comment]
I would recommend to move your dev environment to aws and use database backups for snapshot management
